corehq/apps/sso/templates/sso/config_errors.html

```diff
@@ -0,0 +1,94 @@
+{% extends login_template %}
+{% load i18n %}
+
+{% block title %}{% trans "Single Sign-On is misconfigured" %}{% endblock title %}
+
+{% block login-content %}
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-xs-12">
+        <div class="reg-form-container sign-in-container">
+          <div class="form-bubble form-bubble-lg">
+            <h2>
+              {% blocktrans %}
+                Single Sign-On
+              {% endblocktrans %}
+            </h2>
+
+            {% if request.idp.is_active %}
+              <p class="lead">
+                {% blocktrans with request.idp.is_active as idp_name %}
+                  It looks like there is an issue with {{ idp_name }}'s configuration.
+                {% endblocktrans %}
+              </p>
+              <p>
+                {% blocktrans %}
+                  We have been alerted of this issue and are working with your
+                  account manager to resolve this problem. Thank you for your
```

Comment:
Will this always be true? Is it possible that even though we have been alerted we missed it in the general alert noise and are not actually working on it with an account manager? If that's true that might not be the best message to send here.

Reply:
i think you are right and i'm being too optimistic here. how about...
![Screen Shot 2021-06-04 at 10 38 36 AM](https://user-images.githubusercontent.com/716573/120829671-4b225300-c523-11eb-8562-a40b6862d013.png)